test/integration/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+project(test.mapget.integration)
+
+# TODO: Figure out a way to do this without assuming free ports.
+set (MAPGET_SERVER_PORT 61852)
```

Comment:
Shouldn't there be a test case where the ports are determined automatically as well?